services/py-api/src/server/exception.py

```diff
@@ -10,6 +8,14 @@ class DuplicateEmailError(DuplicateKeyError):
 class DuplicateTeamNameError(DuplicateKeyError):
     pass
 
+
 class HackathonCapacityExceededError(Exception):
     """Exception raised when hackathon capacity has been reached."""
-    pass
\ No newline at end of file
+
+
+class ParticipantNotFound(Exception):
+    """Exception raised when there are no participants that match the query to the database"""
+
+
+class TeamNotFound(Exception):
```

Comment:
TeamNotFoundError

Reply:
Fixed in e8325c7e87aa6e02ae26427cc415c57fdccaa0b3